care/apps/accounts/fixtures/users_fixture_test.csv

```diff
@@ -0,0 +1,49 @@
+id,username,name,email,password,phone_number,is_superuser,user_type,is_staff,created_at,updated_at

```

Comment:
Update email to
Please use the short form for each email.
Eg:

-  root@example.com
- fm11@example.com
etc